modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/ColumnStatisticsCollector.java

```diff
@@ -259,7 +260,8 @@ public static ColumnStatistics aggregate(Comparator<Value> comp, List<ColumnStat
         int averageSize = averageSize(totalSize, total, nullsCnt);
 
         return new ColumnStatistics(min, max, nullsPercent(nullsCnt, total),
-                cardinalityPercent(nullsCnt, total, hll.cardinality()), total, averageSize, hll.toBytes(), ver);
+            cardinalityPercent(nullsCnt, total, hll.cardinality()), total, averageSize, hll.toBytes(), ver,
+            U.currentTimeMillis());
```

Comment:
I don't quite agree. What do you think about `max(partStat.createdAt())`?

Reply:
agree